### DISCUSSION TITLE
macos-deps-arm64-2021-12-05.tar.gz not found

### DISCUSSION THREAD

**Discussion by toruishihara** [obsproject/obs-studio#5657] [Development Questions]
I run full-build-macos.sh, but it end with this error : 
    tar: Error opening archive: Unrecognized archive format
    + ERROR during build step: obs-deps

The script, full-build-macos.sh, try to download tar file from https://github.com/obsproject/obs-deps/releases/download/2021-12-05/macos-deps-arm64-2021-12-05.tar.gz. But that file does not exist on the URL.

**Comment by Fenrirthviti** [comment]
Please use the forums or Discord for assistance with building OBS. Thanks!

https://obsproject.com/help

**Comment by SongAdonis** [comment]
git pull
